x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -546,7 +546,7 @@ processors:
   - dissect:
       if: "ctx._temp_.cisco.message_id == '716002'"
       field: "message"
-      pattern: "Group %{} User %{source.user.name} IP %{source.address} WebVPN session terminated: User Requested."
+      pattern: "Group %{} User %{source.user.name} IP %{source.address} WebVPN session terminated: %{}"
```

Comment:
any value in mapping this to `event.reason`?

Reply:
That sounds good. I was looking for a field to put this into.